features/events/traps/src/test/java/org/opennms/netmgt/trapd/TrapdListenerBlueprintIT.java

```diff
@@ -66,7 +75,19 @@
 	private final TrapNotificationLatch m_handler = new TrapNotificationLatch(3);
 
 	private int m_port;
-
+	
+        private TrapdConfigBean trapConfiguration;
```

Comment:
Move these private fields into each test method. Reusing fields between tests is a bad design.